.gitignore

```diff
@@ -14,9 +14,12 @@ Thumbs.db
 # Project specific
 /mnt/*.json
 /mnt/*.backup
+/mnt/widget*
```

Comment:
why do we need this entry?

Reply:
When we're using ssh keys for authorization, the JSch library, which manages the connection, requires us to pass the path to the file with key. Since we're receiving it from front-end as a field in json object, we have to save it to file before using it. The key used by a widget is saved to a file with widget id as its name. To prevent accidental upload of such data to repository, this entry was added